### PR TITLE
Provide formula for rexster.

### PR DESCRIPTION
**Pull request by splatch**
Formula for rexster: https://github.com/tinkerpop/rexster

### CODE REVIEW

**Comment by tdsmith** [comment]
See audit notes: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/22055/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_rexster___strict/

We follow the [Github Ruby style guide](https://github.com/styleguide/ruby) for new formulas.

`datomic.rb` is an example of a Java formula that installs to `libexec`.

**Comment by splatch** [comment]
In this particular case I followed Cassandra formula which installs everything in #{prefix} avoid nesting everything in libexec.

File: Library/Formula/rexster.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url 'http://tinkerpop.com/downloads/rexster/rexster-server-2.6.0.zip'
+  sha1 '756e317dc4db1b1627b250f3442405518bd9fc6b'
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+
+    prefix.install_metafiles
+    prefix.install Dir["*"]
+  end
+
```
Comment (mikemcquaid) [inline Library/Formula/rexster.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mikemcquaid** [comment]
Please fix the following audits: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/22055/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_rexster___strict/

**Comment by tdsmith** [comment]
That may or may not be right for Cassandra but we shouldn't do that here, sorry. If formula x installs some_dependency.jar to lib and then formula y installs some_dependency.jar to lib, it'll cause link conflicts when Homebrew tries to link them into /usr/local/lib. libexec is a private path for each formula because it's not linked into /usr/local and so we should use it for dependency jars.

File: Library/Formula/rexster.rb (hunk 1 above)
Comment (splatch) [inline Library/Formula/rexster.rb]: Sure - I will add version check as suggested.

**Comment by splatch** [comment]
I didn't know that #{prefix}/lib entries are being auto-linked to /usr/local/lib - my fault. I checked again cassandra formula and it does not have lib/ folder - that's why for this formula it's fine.

I do not want to bring another discussion here but JARs linked in /usr/local/lib would cause troubles only if they would be used by by any other Java formula - which AFAIK doesn't take place. I understand this would cause terrible issues with most of Java formulas.

**Comment by splatch** [comment]
Formula is valid and ready to merge.

**Comment by splatch** [comment]
thx!

**Pull request closed without merging**